src/Plugin/Components/ChatContainer.ts

```diff
@@ -370,28 +415,30 @@ export class ChatContainer {
 					response.map((url) => {
 						content += `![created with prompt ${this.prompt}](${url})`;
 					});
-					this.messages.push({
-						role: assistant,
```

Comment:
Trivial nitpick - but the non string version references a constant that is equal to "assistant"

Reply:
ah. okay I was wondering why this wasn't a string when I was looking at it and working on debugging and just tossed the double quotes in hoping it would solve all my problems. Will revert